components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.html

```diff
@@ -18,19 +18,19 @@ <h2 slot="title">Create {{ objectNoun | titlecase }}</h2>
           <chef-form-field>
             <label>
               <span class="label">ID <span aria-hidden="true">*</span></span>
-              <input chefInput formControlName="id" type="text" (keyup)="handleIDInput($event)" id="id-input">
+              <input chefInput formControlName="id" type="text" (keyup)="handleIDInput($event)" id="id-input" autocomplete="off"/>
```

Comment:
@susanev are there any inputs where we want `autocomplete` on? we could set this by default on the `chef-input` component. (and then use `<chef-input>` here instead of `<input chefInput>`)

Reply:
none that i am aware of.

Reply:
@srenatus in that case, if you want to go a bit further, you could update [chef-input](https://github.com/chef/automate/blob/master/components/chef-ui-library/src/atoms/chef-input/chef-input.tsx) to set `auto-complete=off` (then turn these into chef-inputs, since i'm not sure why they're using the `chefInput` directive in the first place)

Reply:
Oh my, that is spot on. Let's do that instead :) :+1:

Reply:
Done ✔️